### ISSUE TITLE
Any entrence is just rct1 version for everyone

### ISSUE THREAD

**Issue by TheClaudeQC** [OpenRCT2/OpenRCT2#15472]
### Operating System

Windows10 64-bit

### OpenRCT2 build

OpenRCT2, v0.3.4.1 (f7d61c6 on develop) provided by GitHub

### Describe the issue

i try to get back whit the rct2 way to write names (whit lowercase letter), but the game don't allow me to do that

### Steps to reproduce

1. get in any park whit open rides (you need to have rct1 letter way to see the bug)
2. check on options
3. deselect the rct letters on ride
4. it dosen't update

### Attachments

![Bac à sable 2021-09-27 14-23-08](https://user-images.githubusercontent.com/83843734/134964923-3f0e3e91-2185-46c5-9587-24db74fa5596.png)

it's been a while it dose this

**Comment by Gymnasiast** [comment]
Your post is almost impossible to understand, and your screenshot does not help at all.

What you mean is “text on ride banners is always shown in upper case, even when deselecting that option”.

**Comment by TheClaudeQC** [comment]
well, the rct1 have all in upper case, but not in rct2, and i activated this option ealier (about in 0.3.2) and now here (0.3.4.1) when i'm trying to unselect it, it's dosen't update, even when realoading open rct2, so i'm stuck whit rct1 way of writeting on the ride/dynamic signs. that is what is happening.

**Comment by seanmajorpayne** [comment]
Let me see if I can rephrase what you are saying. Please confirm if this is correct.

When you open the Options -> Controls and Interface -> Themes menu, you have 'Roller Coaster Tycoon 2' selected. However, your text on the Entrance ride banner is still using the RCT1 styling with All caps for the lettering. And you are opening the "Edit themes..." menu, and in the features section, you are trying to turn off the RCT1 text styling, but it is not working. So the end result is that you have RCT1 styling on the ride Entrance text, instead of RCT2.

Is this correct?

**Comment by TheClaudeQC** [comment]
> Let me see if I can rephrase what you are saying. Please confirm if this is correct.
> 
> When you open the Options -> Controls and Interface -> Themes menu, you have 'Roller Coaster Tycoon 2' selected. However, your text on the Entrance ride banner is still using the RCT1 styling with All caps for the lettering. And you are opening the "Edit themes..." menu, and in the features section, you are trying to turn off the RCT1 text styling, but it is not working. So the end result is that you have RCT1 styling on the ride Entrance text, instead of RCT2.
> 
> Is this correct?

yes, it's exacly what is happening

**Comment by Gymnasiast** [comment]
You need to toggle this option, then:

![afbeelding](https://user-images.githubusercontent.com/1478678/135766912-0ca1f268-4fe0-49da-95df-ad233f23b53e.png)

**Comment by Makke94** [comment]
Are you supposed to be able to change this while playing? I was testing this and found some buggy behavior while changing the option above while in-game. With the text either not changing to upper/lower-case or flickering between the two.

**Comment by Gymnasiast** [comment]
> Are you supposed to be able to change this while playing? I was testing this and found some buggy behavior while changing the option above while in-game. With the text either not changing to upper/lower-case or flickering between the two.

Yes, but that _used_ to work without a hitch. I initially thought that what you’re describing here was the issue that Claude was trying to report. I should perhaps create an issue of its own for this.

**Comment by Makke94** [comment]
> Yes, but that _used_ to work without a hitch. I initially thought that what you’re describing here was the issue that Claude was trying to report. I should perhaps create an issue of its own for this.

Yeah, not the same issue. Just something I discovered while checking this one out.
